tests/template_tests/test_i18n_tags.py

```diff
@@ -0,0 +1,360 @@
+            )
+            rendered = t.render(Context())
+            self.assertEqual(rendered, '2 andere Super-Ergebnisse')
+
+            # Using 'with'
+            t = Template(
+                '{% load i18n %}{% blocktrans with num_comments=5 context "comment count" %}'
+                'There are {{ num_comments }} comments{% endblocktrans %}'
+            )
+            rendered = t.render(Context())
+            self.assertEqual(rendered, 'Es gibt 5 Kommentare')
+            t = Template(
+                '{% load i18n %}{% blocktrans with num_comments=5 context "other comment count" %}'
+                'There are {{ num_comments }} comments{% endblocktrans %}'
+            )
+            rendered = t.render(Context())
+            self.assertEqual(rendered, 'Andere: Es gibt 5 Kommentare')
+
+            # Using trimmed
+            t = Template(
+                '{% load i18n %}{% blocktrans trimmed %}\n\nThere\n\t are 5  '
+                '\n\n   comments\n{% endblocktrans %}'
+            )
+            rendered = t.render(Context())
+            self.assertEqual(rendered, 'There are 5 comments')
+            t = Template(
+                '{% load i18n %}{% blocktrans with num_comments=5 context "comment count" trimmed %}\n\n'
+                'There are  \t\n  \t {{ num_comments }} comments\n\n{% endblocktrans %}'
+            )
+            rendered = t.render(Context())
+            self.assertEqual(rendered, 'Es gibt 5 Kommentare')
+            t = Template(
+                '{% load i18n %}{% blocktrans context "other super search" count number=2 trimmed %}\n'
+                '{{ number }} super \n result{% plural %}{{ number }} super results{% endblocktrans %}'
+            )
+            rendered = t.render(Context())
+            self.assertEqual(rendered, '2 andere Super-Ergebnisse')
+
+            # Mis-uses
+            with self.assertRaises(TemplateSyntaxError):
+                Template('{% load i18n %}{% blocktrans context with month="May" %}{{ month }}{% endblocktrans %}')
+            with self.assertRaises(TemplateSyntaxError):
+                Template('{% load i18n %}{% blocktrans context %}{% endblocktrans %}')
+            with self.assertRaises(TemplateSyntaxError):
+                Template(
+                    '{% load i18n %}{% blocktrans count number=2 context %}'
+                    '{{ number }} super result{% plural %}{{ number }}'
+                    ' super results{% endblocktrans %}'
+                )
+
+    @override_settings(LOCALE_PATHS=[os.path.join(here, 'other', 'locale')])
+    def test_bad_placeholder_1(self):
+        """
+        Error in translation file should not crash template rendering
+        (%(person)s is translated as %(personne)s in fr.po)
+        Refs #16516.
+        """
+        with translation.override('fr'):
+            t = Template('{% load i18n %}{% blocktrans %}My name is {{ person }}.{% endblocktrans %}')
```

Comment:
Is there a reason why there are still tests for {% trans %} and {% blocktrans %} in here as opposed to putting them in the other tag-specific test files?